app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -230,7 +323,7 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
               "SurfaceProblem" -> averagedStreetFeatures(3)
             )
           )
-          Json.obj("type" -> "Feature", "geometry" -> polygon, "properties" -> properties)
+          Json.obj("type" -> "Feature", "geometry" -> polygon, "properties" -> properties)              
         } else {
           val properties = Json.obj(
             "region_id" -> neighborhood.regionId,
```

Comment:
'I'm realizing that there is no "coverage" property being passed along if the coverage is 0. Could you make sure a coverage of 0.0 is being sent here?